src/SIM/SIMinput.C

```diff
@@ -740,7 +740,7 @@ FunctionBase* SIMinput::parseDualTag (const TiXmlElement* elem, int ftype)
     ftype = 3;
     depth = (Xd.second-Xd.first).length();
   }
-  else if (utl::getAttribute(elem,"u0",xi[0])|
+  else if (static_cast<int>(utl::getAttribute(elem,"u0",xi[0]))|
```

Comment:
Maybe a better solution would be to change the return value of `utl::getAttribute()` to `int`, so this casting would not be needed. And the above ModelGenerator changes would not be needed either (an possibly other/future similarities).